blackbox/docs/admin/cluster-management.rst

```diff
@@ -0,0 +1,101 @@
+.. _cluster_management:
+
+==================
+Cluster management
+==================
+
+With the ``crate-node`` CLI tool you can manage your CrateDB clusters by forming
+new clusters, repairing broken clusters, and by repairing broken nodes.
+
+The ``crate-node`` CLI tool
```

Comment:
Change this to "Introduction" like in https://crate.io/docs/crate/reference/en/latest/run.html